docs/units/logarithmic_units.rst

```diff
@@ -106,18 +106,18 @@ the first star has a known ST magnitude, so we can calculate zero points::
 
     >>> b_ref, v_ref = 17.2 * u.STmag, 17.0 * u.STmag
     >>> b_ref, v_ref  # doctest: +FLOAT_CMP
-    (<Magnitude 17.2 mag(ST)>, <Magnitude 17. mag(ST)>)
+    (<Magnitude 17.2 mag(STflux)>, <Magnitude 17. mag(STflux)>)
```

Comment:
Seeing this in the documentation makes me dislike it more - let's keep `ST`...

Reply:
I agree in principal, but the problem is that then we have a unit called "ST" and "AB" floating around, which is not really meaningful. Are we willing to accept that it's called `mag(ST)` but it's actually `STflux` that's contained inside, @mhvk ?  I.e., have `ST` and `STflux` be aliases or at least exactly equivalent?  Or is that more confusing than this is ugly.

Alternatively we could special-case this and just have the `__repr__` and `__str__` do some tricks to change "STflux" to "ST", although that might be even more confusing.

Reply:
Is it really a problem to have `ST` and `AB` floating around as abbreviations for `STflux` and `ABflux`? We already have `Bol`...

Reply:
I know, but I don't like it - seems ambiguous to me - but Bol at least doesn't have a meaning, whereas I think a lot of people would see `AB` and thing "magnitudes" not "flux".  Hence my concern. 

I guess the core of it here is that mag(ST) "looks like" it's saying "magnitude in the ST system", but really it's "magnitude relative to a fixed zero-point flux of a given value".  That's subtly not the same thing, and actually I think it's good to make it clear that our "magnitude systems" are only valid for specific flux-defined quantites. Hence the need for the maggies-style equivalency in the first place I suppose.